lib/blacklight/solr/response/params.rb

```diff
@@ -15,17 +15,7 @@ def params
     header['params'] || request_params
   end
 
-  def start
-    search_builder&.start || single_valued_param(:start).to_i
-  end
-
-  def rows
-    search_builder&.rows || single_valued_param(:rows).to_i
-  end
-
-  def sort
-    search_builder&.sort || single_valued_param(:sort)
-  end
+  delegate :start, :rows, :sort, to: :search_builder
```

Comment:
Doesn't this require search_builder to exist? I think it can still be nil. Should the initializer for B::S::Response create one if it's missing from the params (ie, if a hash or nil is passed as second arg)?